.hopper/config.yml

```diff
@@ -1,19 +1,19 @@
 version: '1'
 
 services:
-  internal-web:
+  web:
```

Comment:
Too ease the transition, and to make sure we don't have to do anything in lockstep with https://github.com/deliveroo/app-infrastructure/pull/778, maybe keep `web`, `internal-web` and `public-web` here, and then remove `internal-web` once 778 is finalized?

Reply:
Ben, the app is only _live_ in staging and absolutely unused from what I can tell. Therefore IMO we can skip the intermediate step.